diff --git a/CMakeLists.txt b/CMakeLists.txt
index 4dd165b0185..b9fb4c81a7d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -106,7 +106,7 @@ include(build_files/cmake/macros.cmake)
 
 blender_project_hack_pre()
 
-project(Blender)
+project(OpenCVLaboratory)
 
 blender_project_hack_post()
 
@@ -1715,7 +1715,7 @@ include(build_files/cmake/packaging.cmake)
 #-----------------------------------------------------------------------------
 # Use dynamic loading for OpenMP
 if(WITH_BLENDER)
-  openmp_delayload(blender)
+  openmp_delayload(OpenCVLaboratory)
 endif()
 
 #-----------------------------------------------------------------------------
