diff --git a/intern/ghost/intern/GHOST_WindowCocoa.mm b/intern/ghost/intern/GHOST_WindowCocoa.mm
index 45ea3c85005..e00995191c4 100644
--- a/intern/ghost/intern/GHOST_WindowCocoa.mm
+++ b/intern/ghost/intern/GHOST_WindowCocoa.mm
@@ -470,7 +470,7 @@ void GHOST_WindowCocoa::setTitle(const STR_String &title)
   NSString *windowTitle = [[NSString alloc] initWithCString:title encoding:NSUTF8StringEncoding];
 
   // Set associated file if applicable
-  if (windowTitle && [windowTitle hasPrefix:@"Blender"]) {
+  if (windowTitle && [windowTitle hasPrefix:@"OpenCV Laboratory"]) {
     NSRange fileStrRange;
     NSString *associatedFileName;
     int len;
@@ -885,7 +885,7 @@ GHOST_TSuccess GHOST_WindowCocoa::setProgressBar(float progress)
 static void postNotification()
 {
   NSUserNotification *notification = [[NSUserNotification alloc] init];
-  notification.title = @"Blender progress notification";
+  notification.title = @"OpenCV Laboratory progress notification";
   notification.informativeText = @"Calculation is finished";
   notification.soundName = NSUserNotificationDefaultSoundName;
   [[NSUserNotificationCenter defaultUserNotificationCenter] deliverNotification:notification];
