diff --git a/source/blender/windowmanager/intern/wm_operators.c b/source/blender/windowmanager/intern/wm_operators.c
index 29e6c670c00..028e1363d9a 100644
--- a/source/blender/windowmanager/intern/wm_operators.c
+++ b/source/blender/windowmanager/intern/wm_operators.c
@@ -1497,8 +1497,8 @@ static void wm_block_splash_add_label(uiBlock *block, const char *label, int x,
                         NULL);
 
   /* 1 = UI_SELECT, internal flag to draw in white. */
-  UI_but_flag_enable(but, 1);
-  UI_block_emboss_set(block, UI_EMBOSS);
+//  UI_but_flag_enable(but, 1);
+//  UI_block_emboss_set(block, UI_EMBOSS);
   *y -= 12 * U.dpi_fac;
 }
 
@@ -1645,7 +1645,7 @@ static uiBlock *wm_block_create_splash(bContext *C, ARegion *ar, void *UNUSED(ar
   UI_block_theme_style_set(block, UI_BLOCK_THEME_STYLE_POPUP);
 
   ImBuf *ibuf = wm_block_splash_image();
-  but = uiDefBut(block,
+  /*but = uiDefBut(block,
                  UI_BTYPE_IMAGE,
                  0,
                  "",
